connectors/redis-connector/src/test/scala/zio/connect/redis/RedisConnectorSpec.scala

```diff
@@ -0,0 +1,39 @@
+package zio.connect.redis
+import zio.Chunk
+import zio.connect.redis.RedisConnector._
+import zio.stream.ZStream
+import zio.test.{ZIOSpecDefault, _}
+
+import java.util.UUID
+
+trait RedisConnectorSpec extends ZIOSpecDefault {
+
+  val redisConnectorSpec = appendKeySpec + delKeySpec
+
+  private lazy val appendKeySpec = {
+    suite("append")(
+      test("append multiple strings") {
+        val key: String = UUID.randomUUID().toString
+        for {
+          result <- ZStream(Append(key, "a"), Append(key, "b")) >>> append
+        } yield assertTrue(result == ())
+      }
+    )
+  }
+
+  private lazy val delKeySpec = {
```

Comment:
Here we should have a check that the value was indeed added before the deletion.
Example for key:
append key
append key1
r1 = get key1
del key1
r2 = get key1
check r1 has the value that was appended & check r2 is none